src/user/save_user.js

```diff
@@ -1,37 +1,11 @@
 const http = require('http')
-const validateCampaign = require('../validate_campaign')
+const validateUser = require('./validate_user')
+const validateCampaign = require('./validate_campaign')
+const encodeUser = require('./encode_user')
 const debug = require('debug')('user')
 
-module.exports = function saveUser(user, { source }) {
+function makeRequest(userData) {
```

Comment:
made a separate method to separate the request logic from the user data handling logic a bit.